src/server/services/registrar.js

```diff
@@ -203,8 +204,43 @@ async function getShortname(offeredTermCode, classSectionID) {
   }
 }
 
+/**
+ * Get crosslisted courses from the registrar (stored as an array of shortnames)
+ *
+ * @param {string} offeredTermCode  Term.
+ * @param {string} classSectionID   ClassID aka SRS.
+ * @returns {Array}   Returns a list of crosslisted courses.
+ */
+async function getCrosslists(offeredTermCode, classSectionID) {
+  const crosslistXml = await axios(
+    `https://webservices.registrar.ucla.edu/SRDB/SRDBWeb.asmx/getConSched?user=${process.env.REG_DB_USER}&pass=${process.env.REG_DB_PASS}&term=${offeredTermCode}&SRS=${classSectionID};`
+  );
+
+  const crosslists = [];
+  xml2js.parseString(crosslistXml.data, (err, result) => {
```

Comment:
I believe axios handles xml parsing automatically. Might make code slightly cleaner

Reply:
I might be wrong but I couldn't find any docs on that...